android/app/src/main/java/com/expensify/chat/customairshipextender/CustomNotificationProvider.java

```diff
@@ -113,9 +118,8 @@ protected NotificationCompat.Builder onExtendBuilder(@NonNull Context context, @
 
     @RequiresApi(api = Build.VERSION_CODES.O)
     private void createAndRegisterNotificationChannel(@NonNull Context context) {
-        NotificationChannelGroup channelGroup = new NotificationChannelGroup(CHANNEL_GROUP_ID, CHANNEL_GROUP_NAME);
+        NotificationChannelGroup channelGroup = new NotificationChannelGroup(NOTIFICATION_GROUP_CHATS, CHANNEL_GROUP_NAME);
         NotificationChannel channel = new NotificationChannel(CHANNEL_MESSAGES_ID, CHANNEL_MESSAGES_NAME, NotificationManager.IMPORTANCE_HIGH);
-        channel.setGroup(CHANNEL_GROUP_ID);
```

Comment:
This is now named `NOTIFICATION_GROUP_CHATS` and it's already passed when we create the group on line 116

Reply:
I'm guessing these are from Android major version updates? will this break backwards compatibility with older phones?

Reply:
Good spot, but no because the function is annotated with `@RequiresApi(api = Build.VERSION_CODES.O)`. We only call it [if the device meets the correct OS version](https://github.com/Expensify/App/pull/19613/files/327d88ad6bb851306c54ac1d41ea61e36b5d7979#diff-71acb72a4c7edf36c8d1fe2923a467a19738d97049b684879a570c0898fde675R79-R81).

Reply:
👍 noice!